api/envoy/admin/v2alpha/certs.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+// [#protodoc-title: Certificates]
+
+// Proto representation of certificate details. Admin endpoint uses this wrapper for `/certs` to
+// display certificate information. See :ref:`/certs <operations_admin_interface_certs>` for more
+// information.
+message Certificates {
+  // List of certificates known to an Envoy.
+  repeated Certificate certificates = 1;
+}
+
+message Certificate {
+
+  // Details of CA certificate.
+  CertificateDetails ca_cert = 1;
+
+  // Details of Certificate Chain
+  CertificateDetails cert_chain = 2;
```

Comment:
Shouldn't these be repeated? I know current /certs only print the 1st certificate in ca_cert and leaf certificate information in cert_chain for each TLS context, but we will need a full cert chain and CA cert?

Reply:
Ok. My goal was to replicate what we have in /certs with subject alt names added. Do you mind creating issue with what you are describing here so that we can add it once it is merged?

Reply:
Yeah I know you're just replicate current behavior of /certs, so make this repeated, and it is fine to only have one when you fill it.

Reply:
yes both